Clarification of the config (#5)
yiisoft/translator-extractor: README.md

return [
    Extractor::class => [
        '__construct()' => [
            'messageReader' => static fn () => new MessageSource($params['yiisoft/translator']['categorySources']),
            'messageWriter' => static fn () => new MessageSource($params['yiisoft/translator']['categorySources']),
        ],
    ],
];
```

> And in file params.php you can config parameters of message source (if needed)

Reviewer: ```suggestion
And in `params.php` file you can configure parameters of a message source:
```

```php
return [
    'yiisoft/yii-console' => [
        'commands' => [
            'translator/extract' => ExtractCommand::class,
        ],
    ],
    'yiisoft/translator-extractor' => [

>         // example with relative path

Reviewer: ```suggestion
        // Using relative path:
```

        'messagePath' => dirname(__DIR__, 5) . '/messages',
         

>         // example with usage alias of application

Reviewer: ```suggestion
        // Usage aliases:
```

        // 'messagePath' => fn (Aliases $aliases) => $aliases->get('@message'), 
    ],
];
```

> **Attention: MessageReader and MessageWriter should be configured for using SAME MessageSource (One folder in PHP MessageSource).**
> **The extractor needs this to work with existing messages.**

Reviewer: ```suggestion
> **Attention**: Both `MessageReader` and `MessageWriter` should be configured for using _the same_ `MessageSource`. The extractor needs it to work with existing messages.
```
